src/ErrorHandling/Abort.cpp

```diff
@@ -1,26 +1,15 @@
 // Distributed under the MIT License.
```

Comment:
This commit seems to do more than a move

Reply:
reworded